python-sdk/tests/databases/test_base_database.py

```diff
@@ -18,6 +18,24 @@ class DatabaseSubclass(BaseDatabase):
     pass
 
 
+def test_openlineage_database_dataset_namespace():
+    """
+    Test the open lineage dataset namespace for base class
+    """
+    db = DatabaseSubclass(conn_id="fake_conn_id")
+    with pytest.raises(NotImplementedError):
+        db.openlineage_dataset_namespace()
+
+
+def test_openlineage_database_dataset_name():
+    """
+    Test the open lineage dataset names for the base class
+    """
+    db = DatabaseSubclass(conn_id="fake_conn_id")
+    with pytest.raises(NotImplementedError):
+        db.openlineage_dataset_name(table=BaseTable)
```

Comment:
Similarly, here you want to mock the method being called to check that openlineage_dataset_name returns the expected value.